modules/ROOT/pages/index.adoc

```diff
@@ -185,6 +245,14 @@ Studio and Design Center provide various batch components that you can use to
 configure batch processes. For more on this topic, see
 xref:batch-processing-concept.adoc[Batch Processors].
 
+== API Gateway
+
+Mule Runtime includes an embedded API Gateway, which enables you to apply a
```

Comment:
I would say `apply security policies to an API, enrich incoming or outgoing messages and add capabilities to an API without having to write any code.`